app/src/main/res/values/strings.xml

```diff
@@ -68,6 +68,8 @@
     <string name="invalid_branch_environment">Choose urban of rural</string>
     <string name="invalid_branch_sex">Choose man or woman</string>
     <string name="invalid_branch_time_in">Select the time of arrival</string>
+    <string name="invalid_branch_time_out" translatable="false">Select the time of departure</string>
```

Comment:
Please add some translations for these texts in the strings.xml file for romanian language.

Reply:
"Ora plecării nu poate fi mai mică decât ora sosirii."